mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -633,17 +652,7 @@ class LogisticRegressionSuite extends MLTest with DefaultReadWriteTest {
   }
 
   test("sparse coefficients in LogisticAggregator") {
-    val bcCoefficientsBinary = spark.sparkContext.broadcast(Vectors.sparse(2, Array(0), Array(1.0)))
```

Comment:
After we center vectors in mlor, we will remove the impl and relative tests of existing `LogisticAggregator`